src/app/register/components/RegisterForm.jsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+import React from "react";
+import { useRouter } from "next/navigation";
+
+export const RegisterForm = () => {
+  const router = useRouter();
+
+  async function handleRegister(e) {
+    e.preventDefault();
+    const name = e.target.name.value;
+    const email = e.target.email.value;
+    const password = e.target.password.value;
+    const res = await fetch("https://eventmakers-api.fly.dev/auth/register", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        name,
+        email,
+        password,
+      }),
+    });
+    console.log(res.json());
```

Comment:
res.json() needs to be awaited.

Make sure you have tested that this function works 👍

Reply:
ok noted

Reply:
done